services/videoService.js

```diff
@@ -0,0 +1,60 @@
+const { videoDao, classesDao } = require("../models")
+
+const getVideo = async ( classId ) => {
+    const searchClass = await classesDao.getClass(classId);
+
+    if ( !searchClass ) {
+        const err = new Error("INVALID_CLASS");
+        err.statusCode = 406;
```

Comment:
데이터 즉, 리소스가 없는 상태이므로 404가 더 적절 할 것 같아요.
```suggestion
        err.statusCode = 404;
```